### DISCUSSION TITLE
Windows Server 2025 support

### DISCUSSION THREAD

**Discussion by Spc0** [valinet/ExplorerPatcher#3081] [Ideas]
Hi,
First of all i would like to thank you for making explorer patcher, this is the best "tweak UI" tool i've seen for windows 11.
So thank you very very much.

I just tested newest insiders build of windows server, which you all can get for free for testing from this page:
https://www.microsoft.com/en-us/software-download/windowsinsiderpreviewserver
https://techcommunity.microsoft.com/t5/windows-server-insiders/announcing-windows-server-preview-build-26085/m-p/4098829

However when i use explorer patcher, it does work, but it doesn't show clock, icons etc...
So it's not fully functional on server 2025.

Explorer patcher Version:  22621.3296.64.2
Windows Server 2025, Version 24H2, OS Build: 26085.1
Language used: English with Slovenian keyboard and Slovenian clock format.

It has Windows 11 UI with 26085.1 version, which is higher OS build version then current stable Windows 11 client.
This means if you can get explorer patcher running on windows server 2025 it will be running on latest client windows 11 even after updates, as Windows 11 client is always lower build than Windows Server 2025.

I use windows server because it has no adds and no junk like client windows has, and i can get it via my university for free once it will reach RTM status.

Windows server is also LTSC which means no surprise updating over night to new version, which breaks all tweaks.
Windows server always stays the same version and only gets security updates and doesn't break apps/tweaks/programs.
These are best features i really like, not like windows 11 when everything breaks after stealth update to new version over night.

So my question is can you make a support for windows server 2025 as it is the same as windows 11, only newer OS build.

Thank you very much.

**Comment by pyrates999** [comment]
EP is not meant to be run on any windows server versions.

**Reply by Kichura** [reply]
> EP is not meant to be run on any windows server versions.

I would like to proof your point wrong, it *can* run on Windows Server:
![win11](https://github.com/user-attachments/assets/a37edbab-cd28-4e52-bbb6-d74cf71ea8f6)
*(Supported since build 26100, This VM currently runs on build 26334 at time of writing this)*

**Reply by CommandDroid** [reply]
'Not meant to be' and 'cannot' are two diferent things. I CAN drink washing up liquid but it's NOT MEANT to be.

**Reply by Kichura** [reply]
doesn't matter in practice.

**Comment by Spc0** [comment]
New information about LTSC / 24H2.
This includes normal Windows 11 and Server 2025, as they have no difference in UI.

https://www.neowin.net/news/microsoft-blocks-even-more-customization-apps-in-windows-11-version-24h2/

**Comment by pyrates999** [comment]
EP is still not meant to be ran on windows server.  If you want to run EP, install windows 10 or 11.

**Comment by Spc0** [comment]
I know it's not meant to be run on windows server, however some of us convert windows server to workstation you have a ton of guides online how to to that.

Tested on Windows Server 2022, and it works flawlessly, so again thank you very much for this awesome program.
![image](https://github.com/valinet/ExplorerPatcher/assets/165874674/29e00381-fc03-4309-a582-05861ecd3d6d)

**Reply by Willie982** [reply]
It looks like you're using an old version of ExplorerPatcher.
Maybe it will work if you upgrade?
